event_model/schemas/event_descriptor.json

```diff
@@ -106,7 +106,7 @@
             },
             "run_start": {
                 "type": "string",
-                "description": "Globally unique ID to the run_start document this descriptor is associtaed with."
+                "description": "Globally unique ID to the run_start document this descriptor is associated with."
```

Comment:
truly awkward wording, How about

Globally unique ID of this run's 'start' document.

Reply:
check

Reply:
I don't see this text in the revision yet: *Globally unique ID of this run's 'start' document.*